src/vivarium/framework/results/context.py

```diff
@@ -151,29 +195,28 @@ def register_observation(
         when: str,
         **kwargs,
     ) -> None:
-        """Add an observation to the context.
+        """Add an observation to the results context.
 
         Parameters
         ----------
         observation_type
-            Class type of the observation to register.
+            Specific class type of observation to register.
```

Comment:
Maybe have the options listed.

Reply:
While I like the idea, imo it's better to leave it undefined so that we don't need to remember to update this as we add/remove types.